src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -184,11 +184,10 @@ public void should_successfully_return_all_envelopes_with_processed_status_for_a
             .andExpect(MockMvcResultMatchers.jsonPath("envelopes[0].id").exists());
 
         assertThat(envelopeRepository.findAll())
-            .hasSize(2)
+            .hasSize(1)
             .extracting("zipFileName", "status")
             .containsExactlyInAnyOrder(
-                tuple("7_24-06-2018-00-00-00.zip", PROCESSED),
-                tuple("8_24-06-2018-00-00-00.zip", METADATA_FAILURE)
+                tuple("7_24-06-2018-00-00-00.zip", PROCESSED)
             );
```

Comment:
Not sure if this assertion belongs in a test for a controller.
Especially for tests for reading envelopes.

Perhaps something for another round of refactoring.